app/src/main/java/org/walleth/data/networks/RinkebyNetworkDefinition.kt

```diff
@@ -17,4 +17,8 @@ class RinkebyNetworkDefinition : BaseNetworkDefinition() {
     override val bootNodes = listOf("enode://a24ac7c5484ef4ed0c5eb2d36620ba4e4aa13b8c84684e1b4aab0cebea2ae45cb4d375b77eab56516d34bfbd3c1a833fc51296ff084b770b94fb9028c4d25ccf@52.169.42.101:30303?discport=30304")
 
     override val genesis = ""
+
+    override val statsUrl = "https://stats.rinkeby.io/"
+
+    override val infoUrl = "https://www.rinkeby.io/"
```

Comment:
in the end both urls point to the same thing - so I think I will rework this a bit after the merge - I think one link might be enough context